src/__tests__/pages/index.test.tsx

```diff
@@ -1,6 +1,7 @@
 /**
  * @jest-environment jsdom
  */
+import React from 'react'
```

Comment:
I had to add this import in a number of places to appease the linting gods. Is there a better way to handle?

Reply:
Yes! You can add (update) this path in `eslintrc.json` in the `overrides` array

Reply:
Updated -- thank you so much!